config.yaml.defaults

```diff
@@ -28,6 +28,8 @@ app:
       component: SourceList
     - path: "/user/:id"
       component: UserInfo
+    - path: "/upload_photometry/:id"
```

Comment:
Can we rename this to bulk_upload_photometry?

Reply:
Or maybe, `upload_photometry_from_csv`? The current photometry handler already lets you upload photometry in bulk

Reply:
this isn't the API endpoint, just the path name for the page that displays the form. still want it changed?

Reply:
Ah, no nevermind